### PR TITLE
Fixes #17162: Add promote to relay endpoint in API documentation

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17162

### CODE REVIEW

File: webapp/sources/api-doc/openapi.src.yml [hunk 1]
```diff
@@ -60,7 +60,15 @@ tags:
 
 
       Data sources plugin configuration.
+  - name: "🧩 Scale out Relay"
+    description: >-
+      **Requires that the `scale-out-relay` plugin is installed on the server.**
+
+
+      Promote node to relay
```
Comment (amousset) [inline webapp/sources/api-doc/openapi.src.yml:68]: Maybe "Manage relays" as it describes the category

File: webapp/sources/api-doc/paths/scale-out-relay/promote.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# SPDX-FileCopyrightText: 2013-2020 Normation SAS
+post:
+  summary: Promote a node to relay
+  description: Promote a node to relay
+  operationId: promoteToRelay
+  parameters:
+    - $ref: ../../components/parameters/node-id.yml
+  responses:
+    "200":
+      description: Change requests information
```
Comment (amousset) [inline webapp/sources/api-doc/paths/scale-out-relay/promote.yml:11]: needs to be modified

File: webapp/sources/api-doc/paths/scale-out-relay/promote.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                type: string
+                description: Result of the request
+                enum:
+                  - success
+                  - error
+              data:
+                type: string
+                description: Success or error message
+                enum:
+                  - 17dadf50-6056-4c8b-a935-6b97d14b89a7
```
Comment (amousset) [inline webapp/sources/api-doc/paths/scale-out-relay/promote.yml:36]: what is this uuid? it should be en example and not an enum, enum describes the list of possible values

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**